src/pycram/datastructures/pose.py

```diff
@@ -617,3 +729,52 @@ def set_rotation(self, new_rotation: List[float]) -> None:
         :param new_rotation: The new rotation as a quaternion with xyzw
         """
         self.rotation = new_rotation
+
+@dataclass
+class PreferredGraspAlignment:
+    """
+    Description of the preferred grasp alignment for an object.
+    """
+    preferred_axis: Optional[AxisIdentifier]
+    with_vertical_alignment: bool
+    with_rotated_gripper: bool
+
+@dataclass(frozen=True)
+class GraspDescription:
```

Comment:
Why are these dataclasses here? They should be part of datastructures/dataclasses or we make a seperate grasp.py

Reply:
ok, circular imports are defenitly a problem but in that case we should make a sperate file for all grasp related stuff